packages/chatbot/src/utils/curl.ts

```
@@ -188,12 +188,12 @@ function appendMultipart(
 
   if (typeof body === 'string') {
 
-    args.push('-F', body)
+    args.push('-F', body);
 
   }
   else if (Array.isArray(body)) {
 
-    body.forEach(x => args.push('-F', x))
+    body.forEach(x => args.push('-F', x));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)